### PR TITLE
Remove the old Artist View Controller

### PR DESCRIPTION
**Pull request by orta**
This was the first view controller ported to react native.

In trying to move the Artwork VC to. use MP instead of Gravity, I have to re-create all the data models from a query instead of assuming its all there. There's a few things in the artist VC that have fields in MP we haven't added - so it's basically time to give it the ✂️

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'cocoapods'
+gem 'cocoapods', '1.6.0.beta.1'
```
Comment (orta) [inline Gemfile]: for Xcode 10 support

File: Artsy_Tests/View_Controller_Tests/Artist/ARArtistViewControllerTests.m [hunk 2]
```diff
@@ -1,241 +0,0 @@
-#import "ARArtistViewController.h"
```
Comment (orta) [inline Artsy_Tests/View_Controller_Tests/Artist/ARArtistViewControllerTests.m]: our tests peaked at 1k, and are only going downhill from there

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by alloy** [comment]
What a moment!

**Pull request merged**